midi/midi/music_21.py

```diff
@@ -4,7 +4,7 @@
 try:
     from decode import get_sequence_of_notes, export_tempo_array
 except ImportError:
-    from .decode import get_sequence_of_notes, export_tempo_array
+    from .decode import get_sequence_of_notes, export_tempo_array  # TODO: talk with Marcin about this
```

Comment:
hello

Reply:
this workaround is nasty and I wanted you to be aware of that

Reply:
What workaround?

Reply:
try: import